src/ngraph/frontend/onnx_import/core/node.hpp

```diff
@@ -78,6 +78,8 @@ namespace ngraph
             const std::string& output(int index) const;
             std::size_t get_outputs_size() const;
 
+            bool is_attribute_present(const std::string& name) const;
```

Comment:
I'd rename it to `has_attribute` since it's shorter, but I don't insist on this.